<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDB</class>
 <widget class="QMainWindow" name="StudentDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudentDB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Student Info</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name   :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>56</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll No :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textRollNo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonView">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>601</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Details</string>
    </property>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
